### ISSUE TITLE
A11y_VisualStudioCodeClient_Sidebar Bar Options_ Screen reader: Voiceover is announcing the setting button as group button.

### ISSUE THREAD

**Issue by kjawale99** [microsoft/vscode#164698]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #Device; #Visual Studio Code Client; #BM-VisualStudioCodeClient-MAC-Oct2022; WCAG4.1.2; #Voiceover; #Screenreader; #Name,Role,Value.

### Environment and OS details:
Visual Studio Code Insiders
Screenreader: Voiceover

### Repro Steps:

1. Open Visual studio code editor 
2. .Using "Command+Shift+X" key navigate to the extension side view bar option.
3. Tab till "Setting" button near any extension.
4. Verify that Voiceover is announcing the setting button as group button.

### Actual Result:
Voiceover is announcing the setting button as group button.

### Expected Result:
Voiceover should announce the button as "Setting button"

### Attachment:

![button group](https://user-images.githubusercontent.com/98745784/197950986-40c9ef7e-39b3-466c-80c5-108bbf23d17e.gif)

**Comment by isidorn** [comment]
We are setting a correct role on the button. This is a VoiceOver bug and we can not control what exactly does VoiceOver read. 
I suggest to file a WebKit issue here https://bugs.webkit.org/

**Comment by kjawale99** [comment]
@isidorn  Can you please provide the sample app so that we can log third party bug.

**Comment by isidorn** [comment]
Same as https://github.com/microsoft/vscode/issues/164695#issuecomment-1296877475
